Check for the Correct Ready State/app.js

```diff
@@ -0,0 +1,11 @@
+var xhr = new XMLHttpRequest();
```

Comment:
Always use `const` when the value doesn't change.

Reply:
Thanks, @lycoangelo, This is now updated.